openfisca_us/variables/gov/states/mo/tax/income/adjusted_gross_income/mo_adjusted_gross_income.py

```diff
@@ -13,9 +13,7 @@ class mo_adjusted_gross_income(Variable):
     )
     defined_for = StateCode.MO
 
-    def formula(tax_unit, period, parameters):
-        agi = tax_unit("adjusted_gross_income", period)
-        subtractions = tax_unit(
-            "mo_qualified_health_insurance_premiums", period
-        )
+    def formula(person, period, parameters):
+        agi = person("irs_gross_income", period)
```

Comment:
Could you also list a reference for this and leave a comment confirming that the result is capped at zero? Federal AGI can be negative.

```suggestion
        gross_income = person("irs_gross_income", period)
```

Reply:
that should probably be removed then, good catch, thanks